### DISCUSSION TITLE
Common editorlike VSCode's keybinding support

### DISCUSSION THREAD

**Discussion by Anutrix** [godotengine/godot-proposals#4121] [Editor]
Currently. a lot of shortcuts in GDScript editor are uncommon. For new Godot users, it would be helpful if there was an option to use a different editor's(e.g, VSCode) key-binds.

Something like https://code.visualstudio.com/docs/getstarted/keybindings inside GodotEditor would be game(editor)-changing.

**Comment by Calinou** [comment]
Can we change the default bindings to be closer to VS Code's instead? We won't be able to always do this due to conflicts with the rest of the editor though.
